cryostatio/cryostat
# build(mvn): add mvn wrapper

# Welcome to Cryostat! 👋
## Before contributing, make sure you have:

**To recreate commits with GPG signature** `git fetch upstream && git rebase --force --gpg-sign upstream/main`
_______________________________________________

See https://github.com/cryostatio/cryostat-reports/pull/140

## Description of the change:
This adds `mvnw` for an easy consistent development environment.

## Motivation for the change:
Ensures developers/contributors can have a common `mvn` version that is the accepted, canonical build toolchain, to eliminate any build issues due to differences in Maven version used.

## How to manually test:
1. `./mvnw clean package` and ensure things still build and test properly. Bonus points for doing this on a clean system that doesn't even have `mvn` installed/in the `$PATH`.
